app/build.gradle

```diff
@@ -88,6 +89,7 @@ dependencies {
       'androidx.test.espresso:espresso-contrib:3.1.0',
       'androidx.test.espresso:espresso-core:3.2.0',
       'androidx.test.espresso:espresso-intents:3.1.0',
+      'com.android.support.test.espresso:espresso-contrib:3.1.0',
```

Comment:
`Optional Nit`: Arrange this in alphabetical order

Reply:
Also is this same as `androidx.test.espresso:espresso-contrib:3.1.0` ??If so, remove this

Reply:
I checked and we can remove this library

Reply:
Done.